embedded/servo_bq/servo_balsaq.cc

```diff
@@ -35,13 +36,16 @@ void ServoBq::init(int argc, char *argv[]) {
   pwm_driver->enable_auto_increment(true);
   servo = std::make_unique<ServoDriver>(
       0, std::move(pwm_driver),
-      "cfg/servo_cfg0.yaml");  // TODO create standard config dir
-  subscriber_ = make_subscriber("servo_channel_" + std::to_string(servo_index));
+      config_path);  // TODO create standard config dir
+  const std::string channel_name = "servo_channel_" + std::to_string(servo_index);
+  std::cout << "channel name " << channel_name << std::endl;
+  subscriber_ = make_subscriber(channel_name);
 }
 
 void ServoBq::loop() {
   SetServoMessage message;
   if (subscriber_->read(message, 1)) {
+    std::cout << "target angle is " << message.target_angle << std::endl;
```

Comment:
Remove debug prints

Also: Should probably check that the angle is within the max range

Reply:
The driver checks the angle range